pyproject.toml

```diff
@@ -100,7 +100,7 @@ omit = [
 
 [tool.coverage.report]
 precision = 2
-fail_under = 98.35
+fail_under = 96.82
```

Comment:
I'm not willing to introduce this feature without testing it.

Reply:
Yes, I just want to see your attitude towards this PR before adding the test. Because I remember you have implemented something similar before.

Reply:
Ah, then let me review and get back to you.